node-repository/src/main/java/com/yahoo/vespa/hosted/provision/persistence/LoadBalancerSerializer.java

```diff
@@ -171,14 +179,20 @@ private static ZoneEndpoint zoneEndpoint(Inspector settingsObject) {
         if ( ! settingsObject.valid()) return ZoneEndpoint.defaultEndpoint;
         return new ZoneEndpoint(settingsObject.field(publicField).asBool(),
                                 settingsObject.field(privateField).asBool(),
+                                SlimeUtils.entriesStream(settingsObject.field(authMethodsField))
```

Comment:
@mpolden Could you verify that slime handles missing keys correctly here during initial deploy?